SPECS/perl-Module-ScanDeps/perl-Module-ScanDeps.spec

```diff
@@ -64,6 +65,12 @@ make %{?_smp_mflags} test
 %{_mandir}/man3/*
 
 %changelog
+* Fri Nov 15 2024 Pawel Winogrodzki <pawelwi@microsoft.com> - 1.35-2
+- Patched CVE-2024-10224.
+
+* Mon Dec 18 2023 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 1.35-1
+- Auto-upgrade to 1.35 - Azure Linux 3.0 - package upgrades
```

Comment:
Can you check how this 3.0 changelog entry got here?

Reply:
I pulled in the version upgrade from 3.0 and preserved the history.

Reply:
Can we discuss? I'd like to understand better